### PR TITLE
Configurable CA bundle and misc cleanup

### PR DESCRIPTION
**Pull request by Nekroze**

### CODE REVIEW

**Review by KieranB-Jumbo — APPROVED** [review]
Looks good to me.

**Review by KieranB-Jumbo — APPROVED** [review]
Looks good to me.

**Review by KieranB-Jumbo — DISMISSED** [review]
Looks good to me.

**Comment by Nekroze** [comment]
updated to a method that should not affect any existing usage of the library at all, support for using custom ca bundle or the host's root trust store supported via a setter method.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 language: php
 php:
-  - 5.3
```
Comment (Nekroze) [inline .travis.yml:3]: this version was failing in travis as the OS running the tests no longer supports this version

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
-  - 5.4
```
Comment (Nekroze) [inline .travis.yml]: for legacy concerns maybe 5.4 should be tested

**Comment by Nekroze** [comment]
squashing commits together

**Review by grke — DISMISSED** [review]

**Comment by Nekroze** [comment]
[Upstream PR](https://github.com/fatzebra/PHP-Library/pull/10) has been merged, mirroring

**Pull request merged**